lib/utils/normalizeOptions.js

```diff
@@ -1,15 +1,70 @@
 'use strict';
 
+const isAbsoluteUrl = require('is-absolute-url');
+const getCompilerConfigArray = require('./getCompilerConfigArray');
+
 function normalizeOptions(compiler, options) {
-  // Setup default value
-  options.serveIndex =
-    typeof options.serveIndex !== 'undefined' ? options.serveIndex : true;
-  options.contentBase =
-    typeof options.contentBase !== 'undefined'
-      ? options.contentBase
-      : process.cwd();
-  options.contentBasePublicPath = options.contentBasePublicPath || '/';
-  options.watchOptions = options.watchOptions || {};
+  // TODO: improve this to not use .find for compiler watchOptions
+  const configArr = getCompilerConfigArray(compiler);
+  const watchOptionsConfig = configArr.find(
+    (config) => config.watch !== false && config.watchOptions
+  );
+  const watchOptions = watchOptionsConfig
+    ? watchOptionsConfig.watchOptions
+    : {};
+
+  const defaultOptionsForStatic = {
+    directory: process.cwd(),
+    staticOptions: {},
+    publicPath: ['/'],
+    serveIndex: { icons: true },
+    // Respect options from compiler watchOptions
+    watch: watchOptions,
```

Comment:
Do we want this breaking change of `watch` being enabled by default? Previously `watchContentBase` was false by default

Reply:
I think it is good what we watch it by default, most of developers enable it by default